package.json

```diff
@@ -64,7 +65,7 @@
     "eslint-plugin-import": "^2.18.2",
     "eslint-plugin-prettier": "^3.1.0",
     "leaked-handles": "^5.2.0",
-    "parcel-bundler": "^1.12.5",
+    "parcel": "^2.0.0-beta.3.1",
```

Comment:
reminder, v2 is in beta but v1 is deprecated so 🤷‍♂️ (normally we would wait till there was a stable release but we aren't left with that option)